metric/tract/rWLKWK.csv

```diff
@@ -1,61 +1,61 @@
-id,y_2011,y_2012,y_2013,y_2014,y_2015,y_2016,y_2017,y_2018
-37063000101,21,20,21,19,22,9,15,58
-37063000102,13,37,105,166,184,211,152,18
-37063000200,64,68,91,56,32,30,34,46
-37063000301,135,70,92,104,135,209,186,110
-37063000302,115,145,73,77,84,86,83,102
-37063000401,16,43,67,74,82,110,84,82
-37063000402,72,15,71,49,87,92,80,101
-37063000500,115,91,95,63,103,91,89,107
-37063000600,32,98,184,183,157,152,150,114
-37063000700,22,35,34,30,43,39,55,90
-37063000900,0,4,22,22,23,34,26,3
-37063001001,17,35,29,12,11,10,1,2
-37063001002,31,45,62,51,130,59,43,47
-37063001100,0,26,37,60,101,82,69,62
-37063001301,40,52,52,44,10,7,8,0
-37063001303,273,243,243,291,255,277,339,393
-37063001304,9,0,0,0,0,0,0,0
-37063001400,35,58,61,47,56,34,6,7
-37063001501,434,467,419,326,239,171,144,234
-37063001502,653,586,661,767,668,722,799,762
-37063001503,197,227,184,189,112,89,93,132
-37063001601,0,0,15,24,24,25,27,11
-37063001603,14,16,13,14,0,0,0,0
-37063001604,0,16,16,18,20,20,0,51
-37063001705,41,31,35,53,41,0,0,23
-37063001706,15,12,15,0,0,0,0,0
-37063001707,4,12,0,0,0,0,0,0
-37063001708,13,0,0,0,0,12,20,22
-37063001709,53,84,147,61,49,48,69,52
-37063001710,22,47,42,68,73,66,63,63
-37063001711,41,58,79,59,57,54,58,22
-37063001801,28,29,27,0,5,5,4,8
-37063001802,10,9,9,8,0,16,16,19
-37063001806,0,0,0,0,0,0,0,0
-37063001807,36,34,30,39,0,0,0,0
-37063001808,0,0,0,18,33,28,27,25
-37063001809,7,7,7,10,0,0,0,0
-37063001900,0,0,0,0,10,10,10,12
-37063002007,89,94,91,111,0,0,0,0
-37063002008,0,0,0,15,18,17,25,18
-37063002009,13,13,0,0,0,34,46,40
-37063002013,45,25,18,18,31,31,18,33
-37063002015,105,157,88,74,63,69,0,0
-37063002016,47,11,11,11,25,14,20,23
-37063002017,13,22,43,42,58,61,50,29
-37063002018,0,11,156,152,200,217,200,140
-37063002019,17,21,0,0,0,0,0,0
-37063002020,22,0,0,0,0,0,0,0
-37063002021,42,38,44,39,4,7,25,20
-37063002022,0,0,10,15,13,21,21,15
-37063002023,49,35,26,9,19,35,34,27
-37063002024,0,0,0,20,17,23,23,15
-37063002025,0,0,0,0,65,51,57,69
-37063002026,61,63,40,42,27,91,55,142
-37063002027,0,0,0,0,0,14,18,27
-37063002028,0,0,0,0,0,13,45,104
-37063002100,0,29,35,44,44,63,19,24
-37063002200,125,164,201,204,220,140,99,129
-37063002300,43,15,33,36,11,10,9,0
-37063980100,0,0,0,0,0,0,0,0
\ No newline at end of file
+"id","y_2011","y_2012","y_2013","y_2014","y_2015","y_2016","y_2017","y_2018","y_2019"
+37063000101,21,20,21,19,22,9,15,58,54
```

Comment:
These numerators are consistently higher than they should be; possible that the wrong variable got pulled for this year?